src/models/User.model.ts

```diff
@@ -33,6 +45,63 @@ const UserSchema: Schema = new Schema<User>(
       select: false,
       minlength: [8, "Please use minimum of 8 characters"],
     },
+    name: {
+      type: String,
+      required: true,
+    },
+    location: {
+      type: String,
+      required: false,
+      default: "",
+    },
+    url: {
+      type: String,
+      required: false,
+      default: "",
+    },
+    description: {
+      type: String,
+      required: false,
+      default: "",
+    },
+    verified: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+    followers_count: {
+      type: Number,
+      required: false,
+      default: 0,
+    },
+    friends_count: {
+      type: Number,
+      required: false,
+      default: 0,
+    },
+    listed_count: {
+      type: Number,
+      required: false,
+      default: 0,
+    },
+    favourites_count: {
+      type: Number,
+      required: false,
+      default: 0,
+    },
+    statuses_count: {
+      type: Number,
+      required: false,
+      default: 0,
+    },
+    created_at: {
+      type: Date,
```

Comment:
```suggestion
      type: String,
```
you use toISOString() for this field